utils/api_requests.py

```
@@ -62,81 +65,109 @@
+    try:
+        # thanks so much to Seriallos (Raidbots) and BinaryHabitat (GoblinStockAlerts) for organizing this data!
+        bonus_ids = requests.get(
+            "https://www.raidbots.com/static/data/live/bonuses.json"
+        ).json()
+    except Exception as e:
+        print(f"Failed to get raidbots bonus ids getting backup from github: {e}")
+        bonus_ids = requests.get(f"{RAW_GITHUB_BACKUP_PATH}/bonuses.json").json()
     return {int(id): data for id, data in bonus_ids.items()}
 
 
 def get_ilvl_items(ilvl=201, item_ids=[]):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)